packages/hub/node-tests/card-service-test.ts

```diff
@@ -2258,6 +2257,7 @@ describe('hub/card-service', function() {
             {
               name: 'puppies',
               includeFields: [
+                'name',
```

Comment:
This was another policy change. I think before the puppy's `name` field was being included because they would get their embedded field set in addition to the things explicitly requested in the rules.  But then there's no way to occlude the embedded field set.

The new policy is that you only get the embedded fields if you don't specify any rules of your own:

```js
let doc = await daddy.serializeAsJsonAPIDoc({
    includeFieldSet: 'isolated',
    includeFields: [ 'puppies' ]
});
```

Once you start controlling the rules for puppies, that's what you get, with nothing else added:

```js
let doc = await daddy.serializeAsJsonAPIDoc({
    includeFieldSet: 'isolated',
    includeFields: [
      {
        name: 'puppies',
        includeFields: ['name']
      }
    ]
});
```